package.json

```
@@ -69,6 +70,7 @@
     "@types/micromatch": "4.0.2",
     "@types/node": "22.13.14",
     "git-pull-run": "1.4.0",
```

Maybe we should also use `latest` here?

```suggestion
    "git-pull-run": "latest",
```

---

I think I like this, but also I think renovate should be updating this and using an actual version number rather than `latest`

I see a [commit](https://github.com/zirkelc/git-pull-run/commit/24220ffab1faf72c6aadc966b7cd1bea6deb23d8) in a separate branch where it's updated to `1.5.0` because `1.6.0` and `1.6.1` haven't met the requirements of being at least 3 days old (per configuration in [renovate.json](https://github.com/zirkelc/git-pull-run/blob/fb1859dc4ab21b23fa745ea0923e162d77fab08f/.github/renovate.json#L8)). I'm actually a bit confused why there are a handful of [branches](https://github.com/zirkelc/git-pull-run/branches) that renovate hasn't merged or opened PRs for.

All that being said, I think `latest` works here since we should always know what's being published and don't really need renovate to handle keeping `git-pull-run` up to date with itself.